projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -469,5 +475,49 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
     this.setFirstPage();
     this.loadingData = false;
   }
+
+  private buildRowSelectForm(): void {
+    this.rowSelectForm = this.fb.group({});
+
+    this.localTableConfig.tableData.forEach((row: any) => {
+      this.rowSelectForm.addControl(
+        `selection_${row[this.localTableConfig.selectBy]}`,
+        this.fb.control(false)
```

Comment:
Hmm well we need to (1) subscribe to the valueChanges of each control so that we can call the `selectionChange` method when rows are selected/deselected and (2) we need to tie each control to a `go-checkbox` within the template.

Is there a way to do those things without a `FormGroup` (my understanding is the `FormBuilder` is just a convenient way to create form groups)?

Maybe I'm misunderstanding your suggestion.